pkg/comp-functions/functions/vshn-postgres-func/loadBalancer.go

```diff
@@ -55,15 +55,15 @@ func AddLoadBalancerIPToConnectionDetails(ctx context.Context, iof *runtime.Runt
 		k8sservice.Spec.Type = v1.ServiceTypeClusterIP
 	}
 
-	if err := iof.Desired.PutIntoObject(ctx, k8sservice, fmt.Sprintf("%s-%s", comp.GetName(), serviceName)); err != nil {
+	if err := iof.Desired.PutIntoObject(ctx, k8sservice, fmt.Sprintf("%s-%s", comp.GetName(), "service")); err != nil {
```

Comment:
Maybe call it `masterservice` or `mainservice`?

If customers request it, we might also expose the slaves/replicas/secondaries in the future as well.

Reply:
I just renamed it to `primary-service` to comply with Your idea and @TheBigLee suggestions